test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -170,6 +172,29 @@ async fn can_get_script_start_offset() {
     assert_eq!(result.value, script_start_offset as u64);
 }
 
+#[tokio::test]
+async fn can_get_script_bytecode_hash() {
```

Comment:
Should there be another test which somehow checks the negative i.e. comparing one bytecode to another should produce unequal / different results? This proves that the call doesn't return the same output each time

Reply:
So it seems like all simple contract calls share the same script bytecode, so it's hard to test against different scripts. I do think it's a good idea that we check that the script is more than a byte long, which ensures that the script has some complexity to it with a unique hash we should expect